packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -226,8 +190,4 @@ TriggerLauncher.propTypes = {
   initialTrigger: PropTypes.object
 }
 
-export default translate()(
-  withClient(
-    withMutations(accountsMutations, triggersMutations)(TriggerLauncher)
-  )
-)
+export default translate()(withClient(TriggerLauncher))
```

Comment:
Should `TriggerLauncher ` be renamed, as it's not really launching the trigger anymore?

Reply:
Yes. FlowProvider ?

Reply:
Sounds good to me!